src/App.js

```diff
@@ -1,25 +1,54 @@
 import React from 'react';
-
 import './App.scss';
+import { Clock } from './components/clock/clock';
+import { Button } from './components/buttons/buttons';
+
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: 0,
+  }
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+  getRandomName = () => {
+    this.setState({
+      clockName: Math.floor(Math.random() * 100),
+    });
+  }
 
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
+  render() {
+    const gettingClock = this.state.isClockVisible
```

Comment:
What is `gettingClock` ?
```suggestion
    const clockComponent = this.state.isClockVisible
```